alpine/latest/Dockerfile

```diff
@@ -0,0 +1,91 @@
+###############################################################
+#
+# Copyright (C) 2019 James Fuller <jim.fuller@webcomposite.com>
+#
+# SPDX-License-Identifier: MIT
+#
+# pinning to Alpine 3.10.2 as base image
+###############################################################
+FROM registry.hub.docker.com/library/alpine:3.10.2 AS builder
+
+###############################################################
+# set build args
+###############################################################
+ARG CURL_RELEASE_TAG=latest
+ARG CURL_GIT_REPO=https://github.com/curl/curl.git
+ARG CURL_CONFIGURE_OPTION
+ARG LABEL_VERSION=1.0.0
+ARG LABEL_NAME=curl
+ARG LABEL_DESC=curl
+
+###############################################################
+# build curl
+###############################################################
+# install deps and use latest curl release source
+RUN \
+    apk add --no-cache \
+       autoconf    \
+       automake    \
+       build-base  \
+       curl-dev    \
+       groff       \
+       libtool     \
+    && \
+    rm -rf "/var/cache/apk/"*
```

Comment:
Not sure this is necessary with --no-cache.

Reply:
right you are ... removed